### ISSUE TITLE
Distribution-specific packages

### ISSUE THREAD

**Issue by scx** [Beep6581/RawTherapee#5462]
Where is the download page for distribution-specific packages (RPM, DEB, etc.)?
http://web.archive.org/web/20190827230458/https://rawtherapee.com/downloads/
Have you decided to support only AppImage?

**Comment by Beep6581** [comment]
Why do users have to jump through hoops to get RawTherapee, why is it not part of official repositories?

**Comment by scx** [comment]
Some distributions provide too old libraries, which makes building packages for fresh software not so easy. This is the case with CentOS 7, in which the `libtiff` library is too old for RawTherapee (_4.0.3_ < _4.0.4_). That's why I provided the COPR repository for that.
A similar situation may apply to Debian 8 Jessie or Ubuntu 16.04 LTS as well.
https://packages.debian.org/search?keywords=rawtherapee&searchon=names&suite=all&section=all
https://packages.ubuntu.com/search?keywords=rawtherapee&searchon=names&suite=all&section=all
As you can see, both distributions contain heavily outdated versions of the program. COPR/OBS/PPA repositories may be very useful in a such case.
That's why I think it is worth to mention about such repos on the official website, for example in the `alternative packages` section.

**Comment by scx** [comment]
See also:
https://github.com/Beep6581/RawTherapee/issues/5113#issuecomment-535356477
https://github.com/Beep6581/RawTherapee/issues/5115#issuecomment-535357648

**Comment by Beep6581** [comment]
Yes, though I'm just venting about distributions which don't have any package dependency problems but still don't ship current RT in their official repos.

As for this issue, I'm thinking of adding a "Linux Repositories" page somewhere to the website, one page with all the info, probably as a link from the Downloads page.
